tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>"The optimal number of clusters per layer can be found via hyperparameter tuning."</p><p> </p><p>Don't think this tip is useful since it seems like a given since all parameters are things to tune in ML.</p><p> </p><p>The rest are bit more specific to compression specifically.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>